services/core/VolttronCentral/tests/test_vc_autoregister.py

```diff
@@ -8,8 +8,8 @@
                                                       VOLTTRON_CENTRAL_PLATFORM)
 from volttrontesting.utils.agent_additions import (add_volttron_central,
                                                    add_volttron_central_platform)
+from services.core.VolttronCentral.tests.vctestutils import APITester
```

Comment:
NO this should  not have services.core in it as those should not be modules!

Reply:
The proper way to do this is to add the vctestutils to the path sys.path which would allow you to reference the vctestutils without the modules...In pycharm you could mark the tests as source to allow it on the path.

Reply:
Reverted. Can/should the pycharm profile be added to the repo?